### PR TITLE
Add example program demonstrating changing chaintree ownership

### PR DESCRIPTION
**Pull request by aknuds1**
Adding another example program demonstrating the `setOwner` API method. Please provide feedback on this one, as I need your input on whether this is a good approach or not.

Solves part of [this](https://trello.com/c/95d33mvw/152-describe-summary-of-setownership-and-usage-more-clearly-in-docs-examples) card.

The idea is to solve the rest of the card by adding documentation in another PR.

### CODE REVIEW

**Comment by aknuds1** [comment]
> I really love the idea of having something in `examples` directory or tutorial to display ownership changes! A few thoughts:
> 
> 1. I think the full scope of "how do I do a full transfer" is functionally here: https://github.com/quorumcontrol/tupelo-js-sdk/blob/master/test/ownership.js#L96-L156 . It seems valuable to demonstrate the different wallets and the related export / import workflow. Now, I'm not sure its worth duplicating all that here, so maybe
Not sure if I understand you 100%, but if you want to see the example more fleshed out, maybe we could do that in a follow-up PR in order to close this one quickly?

> 2. Definitely would like to see a bit more of the layman's terms description jotted down as well - probably can copy some of the whitepaper's language. Just some general things like: what ownership of a tree means, use of multiple owners, new owner needs the chaintree data somehow.
Where should I add this, in the root README of the project?

> 3. Also might be valuable to add a brief sentence [here](https://quorumcontrol.github.io/tupelo-js-sdk/class/lib/tupelo.js~TupeloClient.html#instance-method-setOwner)
Thanks - what do you want to add here, the kind of documentation from #2? I was in any case planning to update documentation in a follow-up PR to keep it simple.

**Comment by aknuds1** [comment]
@brandonwestcott I rewrote the example app, as I read the `setOwner` tests more closely and understood more of how the functionality works. I also improved the source documentation of `setOwner` a bit as I think it's more correct to say that you define the set of owning keys rather than add to it. Please have another look.

I also added a section to the main README documenting the existence of the example apps - I think this should help users finding documentation on the `SET_OWNERSHIP` feature.

I think this should solve the [card](https://trello.com/c/95d33mvw/152-describe-summary-of-setownership-and-usage-more-clearly-in-docs-examples) in question, but please let me know if there's more documentation I should add as I would eventually need input.

File: example/transfer-ownership/README.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+First you must ensure that you have installed Node.js and a Tupelo RPC server is
+listening on "localhost:50051". See the Tupelo.js README for more details if
+necessary before running this sample program.
+
+NOTE: restarting the local RPC server will wipe out all data, therefore existing registered
+chains will no longer work. Either remove the `.wallet-state` file to get a clean slate, or
+simply register a new chain tree.
+
+#### Usage
+Once the RPC is running you can run three commands with ownership.js. After registering a source
```
Comment (cap10morgan) [inline example/transfer-ownership/README.md]: Add "server" after "RPC" here?

**Comment by aknuds1** [comment]
Thanks @cap10morgan! I fixed both example READMEs with regards to "RPC server".

**Pull request merged**